grommet-leaflet/src/layers/MarkerCluster/index.d.ts

```diff
@@ -0,0 +1,17 @@
+import * as React from 'react';
+import * as L from 'leaflet';
+import 'leaflet.markercluster';
+
+export interface MarkerClusterProps {
+  icon?: () => any;
```

Comment:
```suggestion
  icon?: JSX.Element;
```

Reply:
Changed to `icon?: (any) => JSX.Element;`